koku/api/dump/models.py

```diff
@@ -0,0 +1,48 @@
+"""Models for user-initiated data dump requests."""
+
+from uuid import uuid4
+
+from django.db import models
+
+
+class DumpRequest(models.Model):
+    """A user's request for a data dump."""
+
+    PENDING = 'pending'
+    COMPLETE = 'complete'
+    STATUS_CHOICES = ((PENDING, 'Pending'), (COMPLETE, 'Complete'))
```

Comment:
Is there some reason three local variables are being assigned here? 

I don't see any use of `PENDING` or `COMPLETE` beyond its use in `STATUS_CHOICES`. So, I think that can be collapsed into just a single variable.

Reply:
This is Django's recommended practice for defining choices. See: https://docs.djangoproject.com/en/2.2/ref/models/fields/#choices

> Generally, it’s best to define choices inside a model class, and to define a suitably-named constant for each value:

Also, they will likely be used in future commits as this feature grows out, and when they are, referencing a class variable will better than proliferating strings.